tooling/ast_fuzzer/src/compare/interpreted.rs

```diff
@@ -160,7 +160,7 @@ impl Comparable for ssa::interpreter::value::Value {
 }
 
 /// Convert the ABI encoded inputs to what the SSA interpreter expects.
-fn input_values_to_ssa(abi: &Abi, input_map: &InputMap) -> Vec<ssa::interpreter::value::Value> {
+pub fn input_values_to_ssa(abi: &Abi, input_map: &InputMap) -> Vec<ssa::interpreter::value::Value> {
```

Comment:
Should this function be somewhere near the interpreter? It's a bit strange that this is in the fuzzing tool and it's called from `nargo_cli`.

Reply:
I agree, as I said in the description, I would follow up with a PR that moves some of the utility functions from the fuzzer to common libraries. Another example is the function that produces an ABI description from a `Program`. 

I just didn't want to make this PR bigger than it has to be.

Reply:
Added https://github.com/noir-lang/noir/issues/8707 to follow up on this.